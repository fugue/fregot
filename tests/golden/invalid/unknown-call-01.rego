package fregot.tests.invalid.unknown_call_01

test_something {
  bar.unknown_call(1)
}
