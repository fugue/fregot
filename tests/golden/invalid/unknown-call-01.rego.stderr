fregot (renamer error):
  "unknown-call-01.rego" (line 4, column 3):
  unknown function:

    4|   bar.unknown_call(1)
         ^^^^^^^^^^^^^^^^^^^

  Package bar is not imported.
